backend/tests/unit/services/test_mcp_server_registry_masking.py

```
@@ -0,0 +1,388 @@
+Unit tests for MCPServerRegistry masking configuration support.
+
+Tests focus on practical masking integration scenarios:
+- Server config retrieval with masking configurations
+- Built-in vs configured server masking support
+- Safe config lookup for masking service integration
+"""
+
+import pytest
+
+from tarsy.models.mcp_config import MCPServerConfig
+from tarsy.models.masking_config import MaskingConfig, MaskingPattern
```

Ruff (0.12.2)

`tarsy.models.masking_config.MaskingConfig` imported but unused

Remove unused import

(F401)